plf-packaging-resources/src/main/resources/README.txt

```diff
@@ -9,13 +9,13 @@ System requirements
     * Memory:  4GB of RAM (8GB recommended)
     * Disk:    2GB (depending of the amount of data)
     * OS:      Windows or Linux
-    * JDK:     Java 7 (Set the JAVA_HOME environment variable)
-    * Browser: Google Chrome 25+, Firefox 19+ or Internet Explorer 10+
+    * JDK:     Java 8 (Set the JAVA_HOME environment variable)
```

Comment:
Do we need a JDK or is JRE sufficient ? I think a JRE is sufficient now that we don't use eXo IDE anymore.

Reply:
We only test with a JDK so I think it is better to keep it like that.

Reply:
ok, we may have to update this for next versions (JRE should be used instead of JDK for tests)